documentation/visualization.md

```diff
@@ -61,8 +61,8 @@ A descriptive name of the Dimension, this description can be multilingual.
 #### schema:description
 A description of the Dimension, this description can be multilingual.
 
-#### qudt:unit
-To describe the unit of the values in a dimension the respective `qudt:Unit` instance can be attached to a [Dimension Constraint](#dimensionconstraints) with the `qudt:unit` property.
+#### qudt:unit / qudt:hasUnit
+To describe the unit of the values in a dimension the respective `qudt:Unit`/ `qudt:hasUnit` instance can be attached to a [Dimension Constraint](#dimensionconstraints) with the `qudt:unit` / `qudt:hasUnit` property.
```

Comment:
beware: `qudt:Unit` (capital U) is used for classes and is out of scope here as it is unaffected by the deprecation of `qudt:unit` in favor of `qudt:hasUnit`. 

Thus you may not alter `qudt:Unit` i.e. 

- To describe the unit of the values in a dimension the respective `qudt:Unit` instance can be attached to a [Dimension Constraint](#dimensionconstraints) with the `qudt:unit` / `qudt:hasUnit` property.

And as you still mention both it might be a good hint that `qudt:unit` will be dropped in favor of `qudt:hasUnit`

Reply:
there has not been a create or concept of our documentation change from Zazuko  but there was a simple execute 1:1 of what was ordered in the Issue about the merge for qudt:hasUnit (point in the 4 mentions to the other prompt) - if there is any other/more inputs/change wishes, please adapt accordingly to what makes sense/is complete in your perspective as I am not the role in charge of the content of this documentation.

Reply:
see https://github.com/zazuko/cube-link/pull/135